frontend/integration-tests/data/caddy-docker.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
It would be better to make test data YAML so it's easier to maintain

Reply:
ok, will update